### ISSUE TITLE
Toolbar overlaps nested when wrapping to new line

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#36]
**Description**
Nested toolbar overlaps when parent toolbar wraps to new.

**Additional info**
- Plugin version: 1.0.2
- Craft version: 3.6.15
- Multi-site: Yes

**Additional context**
<img width="629" alt="Screenshot 2021-05-31 at 19 26 29" src="https://user-images.githubusercontent.com/14869578/120225757-1c1b9100-c246-11eb-94f0-74e2a38c1ff9.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
